source/creatureeffect/CreatureEffectSpeedChange.cpp

```diff
@@ -59,6 +59,7 @@ void CreatureEffectSpeedChange::applyEffect(Creature& creature)
 
 void CreatureEffectSpeedChange::releaseEffect(Creature& creature)
 {
+    //FIXME: sanitizer reports that this is called at least once on something that is not a creature.
```

Comment:
Yeah should probably create an issue for stuff reported by the sanitizer. I'm not sure if I would call the issue false positives, as undefined behaviour can have unexpected effects.